.eslintrc.md.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  extends: [
+    "plugin:markdownlint/recommended",
+  ],
+  "parser": "eslint-plugin-markdownlint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    },
+  },
+  rules: {
```

Comment:
@m4sterbunny all the various rules to get things going. Have fixed up the bits in md/mdx as well so it now runs as part of the lint step

Reply:
@joaniefromtheblock FYI Josh has setup some of the rules to get ES link going